rules/windows/persistence_runtime_run_key_startup_susp_procs.toml

```diff
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2020/11/19"
+maturity = "production"
+updated_date = "2020/11/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies execution of suspicious persistent program (Scripts, Rundll32, etc.) by looking at process lineage and
+command line usage. In order to survive reboots and other system interrupts, attackers will modify run keys within the
+registry or leverage startup folder items as a form of persistence.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Execution of Persistent Suspicious Program"
+risk_score = 43
+rule_id = "e7125cea-9fe1-42a5-9a05-b0792cf86f5a"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+type = "eql"
+
+query = '''
+/* userinit followed by explorer followed by early child process of explorer (unlikely to be launched interactively) within 1m */
+sequence by host.id, user.name with maxspan = 2m
+  [process where event.type in ("start", "process_started") and process.name : "userinit.exe" and process.parent.name : "winlogon.exe"]
+  [process where event.type in ("start", "process_started") and process.name : "explorer.exe"]
+  [process where event.type in ("start", "process_started") and process.parent.name : "explorer.exe" and
+   /* add suspicious programs here */
+   process.pe.original_file_name in ("cscript.exe",
+                                     "wscript.exe",
+                                     "PowerShell.EXE",
+                                     "Cmd.Exe",
+                                     "MSHTA.EXE",
+                                     "RUNDLL32.EXE",
+                                     "REGSVR32.EXE",
+                                     "RegAsm.exe",
+                                     "MSBuild.exe",
+                                     "InstallUtil.exe") and
+    /* add potential suspicious paths here */
+    process.args : ("C:\\Users\\*", "C:\\ProgramData\\*", "C:\\Windows\\Temp\\*", "C:\\Windows\\Tasks\\*", "C:\\PerfLogs\\*", "C:\\Intel\\*")
+   ]
+'''
+
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1060"
+name = "Registry Run Keys / Startup Folder"
```

Comment:
doesn't really align to this rule

Reply:
disagree, any program set to persist via run keys or startup at runtime is a child of explorer.exe (straight after user session setup "userinit").

Reply:
what about runonce.exe? there's a bit of both

Reply:
runonce.exe wasn't in initial rule, also if set under HKEY_USERS it will defaults to explore.exe but if set under HKLM it will default to a child of runonce.exe (can be added as different rule, won't fit in the sequence and also rarely attackers will drop a persistence in runonce under HKLM, they have better options with admin priv anyway).